qa/integration/src/test/resources/features/sso/SsoLoginUsingGenericProvider.feature

```diff
@@ -0,0 +1,18 @@
+###############################################################################
+# Copyright (c) 2020 Eurotech and/or its affiliates and others
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     Eurotech - initial API and implementation
+###############################################################################
+@sso
+@integration
+
+Feature: Sso Login using Generic Provider
+
+  Scenario: Configure environment for SSO with the Generic provider
+    Given Set environment variables for generic provider
```

Comment:
Hi @zanpizmoht , I think that the tests scenarios defined in `SsoLoginUsingKeycloakProvider.feature` should be added also here, in order to carry the tests also with the generic provider. I'm not 100% sure, but maybe we can find a way to reuse the same scenarios without simply copying the code here.

Reply:
I will look into the tests for generic provider when this PR will be merged.